av1an-core/src/encoder.rs

```diff
@@ -25,6 +26,19 @@ pub enum Encoder {
   x265,
 }
 
+pub static USE_OLD_SVT_AV1: Lazy<bool> = Lazy::new(|| {
+  let version = Command::new("SvtAv1EncApp")
+    .arg("--version")
+    .output()
+    .unwrap();
+
+  if memchr::memmem::find(b"v0.9.0", &version.stdout).is_some() {
```

Comment:
Clippy says "help: you can reduce it to: `!memchr::memmem::find(b"v0.9.0", &version.stdout).is_some()`"